pkg/output/printer.go

```diff
@@ -43,27 +41,35 @@ const (
 )
 
 type PrintOptions struct {
-	Fields      []string
-	FieldsLong  []string
-	IgnoreFlags bool
-	Output      OutputOption
-	Pager       io.Writer
-	NoPager     bool
-	NoHeader    bool
-	Separator   string
+	// Fields is a list of fields to print
+	Fields []string
```

Comment:
Is it impossible to control what secondary fields get printed? Does this apply to JSON?

Reply:
i guess so if i understand the question correctly

- When user runs `tctl workflow list`, it prints the fields listed in `PrintOptions.Fields`
- `tctl workflow list --fields long` prints fields PrintOptions.Fields + PrintOptions.FieldsLong
- `tctl workflow list --fields Field1,Field2.NestedField3` prints user fields 

`--fields long` seems like what you refer to by the secondary fields?

Reply:
json view ignores this and prints the whole structure